src/Redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+const CHECK_STATUS = 'checkStatus';
+
```

Comment:
- [ ] Kindly ensure that you set the initial state to be an empty array of categories also.